Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -47,26 +44,26 @@ protected override void Awake()
                     RegisterPointer(pointingSource);
                 }
             }
-        }
 
-        private void Start()
-        {
-            if (pointers.Count == 0 && autoRegisterGazePointerIfNoPointersRegistered && GazeManager.IsInitialized)
+            if (pointers.Count == 0 && autoRegisterGazePointerIfNoPointersRegistered && GazeManager.CofirmInitialized())
```

Comment:
I change a good deal of things here in the focus manager concerning pointers. Probably not a good use case example.